assets/operatingInfo/aselo-dev.json

```diff
@@ -0,0 +1,26 @@
+{
+  "timezone": "America/New_York",
+  "holidays": { 
+    "12/25/2021": "Christmas"
+  },
+  "operatingHours": {
+    "webchat": {
+      "1": [{ "open": 0, "close": 2359 }],
```

Comment:
If I remember right, the logic for close was a strict `<`, not `<=`.  So this may return `closed` at 11:59pm.

Reply:
You are right, thanks for the catch!